yt_dlp/extractor/rtp.py

```diff
@@ -8,7 +8,7 @@
 
 
 class RTPIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?rtp\.pt/play/p(?P<program_id>[0-9]+)/(?P<id>[^/?#]+)/?'
+    _VALID_URL = r'https?://(?:www\.)?rtp\.pt/play/(?:zigzag/)?p(?P<program_id>[0-9]+)/(?P<id>[^/?#]+)/?'
```

Comment:
Can you add `estudoemcasa` and `palco` as well?
```suggestion
    _VALID_URL = r'https?://(?:www\.)?rtp\.pt/play/(?:(?:estudoemcasa|palco|zigzag)/)?p(?P<program_id>[0-9]+)/(?P<id>[^/?#]+)'
```
Please also add appropriate tests. match_only is fine.
 - https://www.rtp.pt/play/estudoemcasa/p7776/portugues-1-ano
 - https://www.rtp.pt/play/palco/p13785/l7nnon